pages.zh/common/jmap.md

```diff
@@ -0,0 +1,20 @@
+# jmap
+
+> Java 内存映射工具。
+> 更多信息：<https://docs.oracle.com/javase/7/docs/technotes/tools/share/jmap.html>.
+
+- 打印 Java 进程的共享对象映射（类似 pmap 的输出）：
+
+`jmap {{Java进程号}}`
+
+- 打印堆摘要信息：
+
+`jmap -heap {{filename.jar}} {{Java进程号}}`
```

Comment:
```suggestion
`jmap -heap {{导出文件名.jar}} {{Java进程号}}`
```
That probably looks weird, but shouldn't that be translated?

Reply:
I will study the code implementation of jmap later, I feel that there are some problems with this parameter, please wait patiently. 😁 @marchersimon

Reply:
```suggestion
`jmap -heap {{文件.jar}} {{Java 进程号}}`
```

Reply:
I read the JMap tool code implementation of JDK7,8 and 9, and found that the {filename.jar} parameter passed in jmap cannot play any role, so I considered removing {filename.jar}

I don’t know where the contributors of the original English version obtained the {filename.jar} parameter. If it is convenient, please introduce the source of this parameter, thanks.

cc @marchersimon 


links
- https://github.com/openjdk/jdk/blob/7b2e99178f7cf41ecd86b2ccfba38ea653e815e7/jdk/src/share/classes/sun/tools/jmap/JMap.java#L151
- https://github.com/openjdk/jdk/blob/7b2e99178f7cf41ecd86b2ccfba38ea653e815e7/hotspot/agent/src/share/classes/sun/jvm/hotspot/tools/JMap.java#L116
- https://github.com/openjdk/jdk/blob/0f428ca533627e758cdca599d4589f39c21e27b6/src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/tools/JMap.java#L58
- https://github.com/openjdk/jdk/blob/b6a729a79b2ca65ec3dc26af1ee8dcd1cd3afa41/hotspot/agent/src/share/classes/sun/jvm/hotspot/tools/JMap.java#L58

Reply:
source page: https://github.com/tldr-pages/tldr/blob/main/pages/common/jmap.md